packages/editor/src/components/Input.js

```diff
@@ -0,0 +1,35 @@
+/** @jsx jsx */
+import { jsx, useColorMode } from 'theme-ui'
+import Field from './Field'
+
+const Input = ({ label, ...props }) => {
+  const [colorMode] = useColorMode()
+  return (
+    <Field label={label}>
+      <input
```

Comment:
Personally I'd rather see the label element composed here – it's unclear whether or not the `<label>` element has a correct `for` attribute

Reply:
Done 👍